queries/ses_create_log_table.sql.tftpl

```diff
@@ -0,0 +1,63 @@
+CREATE EXTERNAL TABLE IF NOT EXISTS ses (
```

Comment:
**issue**: created table is named `ses.ses` (`$databse.$table`). All queries go against `ses.ses_bounce_logs`. Rename here or in all queries.